realm/realm-library/src/androidTest/java/io/realm/rule/TestRealmConfigurationFactory.java

```diff
@@ -109,4 +116,20 @@ public RealmConfiguration createConfiguration(String name, byte[] key) {
     public RealmConfiguration.Builder createConfigurationBuilder() {
         return new RealmConfiguration.Builder(getRoot());
     }
+
+    // Copies a Realm file from assets to temp dir
+    public void copyRealmFromAssets(Context context, String realmPath, String newName)
```

Comment:
This is also in `TestHelper`, but I agree it makes more sense here. Can we delete it outright from `TestHelper` or do we need to deprecate it first until all classes have moved to use `TestRealmConfigurationFactory`?